docs/source/install/install.rst

```diff
@@ -395,6 +349,7 @@ The following browsers are unsupported:
 * Chrome < 13
 * Opera (any): CSS issues, but execution might work
 * Internet Explorer < 10
+* Internet Explorer ≥ 10 (same as Opera)
 
 The following specific combinations are known **NOT** to work:
```

Comment:
I don't think we should bother listing these. They're probably out of date anyway.